vaadin-grid-flow-integration-tests/src/test/java/com/vaadin/flow/component/grid/it/SortingIT.java

```diff
@@ -61,6 +62,57 @@ public void setInitialSortOrder_changeOrderFromServer_sortIndicatorsUpdated() {
         findElement(By.id("sort-by-age")).click();
         assertAscendingSorter("Age");
     }
+    @Test
+    public void keepSortStatesAfterReAttach() {
+        WebElement btnAttach = findElement(By.id("btn-attach"));
+        WebElement btnRemove = findElement(By.id("btn-detach"));
+        GridElement sortingGridElement = $(GridElement.class).first();
+        findElement(By.id("sort-by-age")).click();
+        // Attach grid
+        btnAttach.click();
+
+        WebElement sortingGrid = findElement(By.id("sorting-grid"));
+        Assert.assertEquals("asc", sortingGrid.findElements(By.tagName("vaadin-grid-sorter"))
+                .get(1).getAttribute("direction"));
+        String textAgeColumnBeforeReattch = sortingGridElement.getCell(0, 1).getText();
+        // Detach
+        btnRemove.click();
+        // Reattach
+        btnAttach.click();
+        sortingGridElement = $(GridElement.class).first();
+        sortingGrid = findElement(By.id("sorting-grid"));
```

Comment:
Isn't `sortingGridElement` and `sortingGrid` the same exact object?

Reply:
The first one, I want to take a row, cell and so on of grid (GridElement class)
The second, I want to take the tag name of vaadin-grid-sorter, from the grid, to limit the range when using findElements. but I removed the second because I think it not necessary. there is only one grid.

Reply:
Yes, the GridElement is also a WebElement